src/components/JobOverview/JobOverview.js

```diff
@@ -46,12 +46,12 @@ const JobOverview = ({
     // set selected job to first job if
     // first time user visits the job overview page
     useEffect(() => {
-        if (!selectedJob && allTasks.length > 0) {
+        if (selectedJob === undefined && allTasks.length > 0) {
             setSelectedJob(allTasks[0])
         }
     }, [])
 
-    if (!selectedJob) {
+    if (allTasks.length == 0) {
```

Comment:
Prefer strict equals unless checking for null or undefined which can be `nullish == null`